xarray/core/indexing.py

```diff
@@ -164,7 +95,20 @@ def get_loc(index, label, method=None, tolerance=None):
 
 
 def get_indexer(index, labels, method=None, tolerance=None):
```

Comment:
Rename this to `get_indexer_nd` to differentiate it from the pandas method `Index.get_indexer`?

Reply:
Done